src/internal/connector/onedrive/collection.go

```diff
@@ -415,6 +415,7 @@ func (oc *Collection) populateItems(ctx context.Context) {
 					ItemName:   itemInfo.OneDrive.ItemName,
 					DriveName:  itemInfo.OneDrive.DriveName,
 					ItemType:   itemInfo.OneDrive.ItemType,
+					IsMeta:     true,
```

Comment:
The filenames are pretty stable, they will always have one of `.data`, `.meta` or `.dirmeta` suffixes. Should we just make use of that instead of adding new filed in the driveInfo? This might be useful long term though.

Reply:
Stability isn't the concern.  Cross-contamination is my worry.  If we back-up a file named `foo.meta` that has nothing to do with our meta scheme, it could cause problems.  Sure, right now it'll be called `foo.meta.data`, but if we end up changing that naming scheme in the future, this might still catch us by surprise.

And/or we can drop it when we move to storing files by ID.